package-res/resources/web/pentaho/visual/base/events/mixinDataFilter.js

```diff
@@ -0,0 +1,52 @@
+/*!
+ * Copyright 2010 - 2016 Pentaho Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+define([], function() {
+  "use strict";
+
+  /**
+   * @name mixinDataFilter
```

Comment:
Documented as a class, but it is just a prototype object.
Why not document as a `@mixin`? And then use `@mixes`?
Why not implement as a Base class?